backend/profiles/models.py

```diff
@@ -1,4 +1,15 @@
 from django.db import models
 from authentication.models import User
-# Create your models here.
+from datetime import datetime
 
+
+class FollowRecord(models.Model):
+    following_user = models.ForeignKey(User, on_delete=models.CASCADE, related_name="user_following")
+    followed_user = models.ForeignKey(User, on_delete=models.CASCADE, related_name="user_followed")
+    follow_date = models.DateTimeField(default=datetime.now())
```

Comment:
As we have noticed today, using `default=datetime.now()` causes a bug where all instances of this field display the same fixed date, the date on which this model was first created on the database. We figured out today that we should be using `auto_now_add=True` instead of a default value.

Reply:
Oh, I also thought `datetime.now()` was problematic, good thing we have a solution. Thanks!